jobs/component_jobs.groovy

```diff
@@ -104,22 +101,20 @@ repos.each { Map repo ->
       }
 
       steps {
-        // if a PR commit, build and push immutable docker tag
-        if (isPR) {
-          shell '''
-            #!/usr/bin/env bash
+        shell '''
+          #!/usr/bin/env bash
 
-            set -eo pipefail
+          set -eo pipefail
 
-            make bootstrap || true
+          make bootstrap || true
+
+          export IMAGE_PREFIX=deisci
+          docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+          DEIS_REGISTRY='' make docker-build docker-push
+          docker login -e="$QUAY_EMAIL" -u="$QUAY_USERNAME" -p="$QUAY_PASSWORD" quay.io
+          DEIS_REGISTRY=quay.io/ make docker-build docker-push
```

Comment:
note this would push both immutable and mutable tags both on PR job and master job -- do we want logic to only push immutable on PR and then both on master?